modules/common/src/main/ts/api/ErrorExtractor.ts

```diff
@@ -0,0 +1,109 @@
+import * as Differ from './Differ';
+import * as TestError from './TestError';
+import * as LoggedError from './LoggedError';
+
+type LoggedError = LoggedError.LoggedError;
+
+type TestError = TestError.TestError;
+type PprintAssertionError = TestError.PprintAssertionError;
+type HtmlDiffAssertionError = TestError.HtmlDiffAssertionError;
+type AssertionError = TestError.AssertionError;
+
+export interface BasicErrorData {
+  readonly type: string;
+  readonly message: string;
+  readonly diff?: {
+    readonly expected: string;
+    readonly actual: string;
+    readonly comparison: string;
+  };
+}
+
+export interface ErrorData extends BasicErrorData {
+  readonly logs?: string;
+  readonly stack?: string;
+}
+
+const stringify = (e: any) => {
```

Comment:
```suggestion
const stringify = (e: unknown) => {
```

Reply:
`any` is the correct type here as per `JSON.stringify` types.